### PR TITLE
Add paragraph translation sync module

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Adds the patch from issue 2887353 to enable new sub module "Paragraph Translation Sync."
Allows per paragraph "status" field, allowing users to hide individual paragraphs for non-default languages.

Enables 'translate paragraph' permission for anonymous users to resolve the visibility issue described here.
https://www.drupal.org/project/paragraphs/issues/2902371#comment-12577532
I tested as anonymous and since there are no 'edit content' permissions this workaround appears safe.

Includes update hook to enable module, update anonymous user config.

There are a couple of minor config updates related to the press releases and publications.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes/no
| Documentation reflects changes? | yes/no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Medium
| Relevant links | https://thinkoomph.jira.com/browse/RIGA-10
https://www.drupal.org/node/2887353

### CODE REVIEW

File: ecms_base/config/install/user.role.anonymous.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ weight: 0
 is_admin: false
 permissions:
   - 'access content'
+  - 'translate paragraph'
```
Comment (JMStewart00) [inline ecms_base/config/install/user.role.anonymous.yml:10]: For some reason this seems off? Anonymous users can translate paragraphs?

File: ecms_base/ecms_base.install [hunk 2]
```diff
@@ -895,3 +895,34 @@ function ecms_base_update_9054(array &$sandbox): void {
...
+  $path = \Drupal::service('extension.list.profile')->getPath('ecms_base');
+
+  /** @var \Drupal\Core\Config\StorageInterface $active_storage */
+  $active_storage = \Drupal::service('config.storage');
+
+  /** @var \Drupal\Core\Config\FileStorage $install_source */
+  $install_source = new FileStorage($path . "/config/install/");
+
+  $updatedConfig = [
+    'user.role.anonymous',
```
Comment (JMStewart00) [inline ecms_base/ecms_base.install:921]: Same kinda question as the other call out.

Comment (ben-hamelin) [reply ecms_base/ecms_base.install:921]: @JMStewart00  Yea, it's an odd work-around but it seemed to solve the issue noted here:
https://www.drupal.org/project/paragraphs/issues/2902371#comment-12577532

I tested and anonymous can't access any edit screens

**Review by JMStewart00 — APPROVED** [review]
Ok looks good

File: ecms_base/ecms_base.install (hunk 2 above)
Comment (JMStewart00) [reply ecms_base/ecms_base.install:921]: oh cool.

**Pull request merged**